src/types/index.ts

```diff
@@ -117,7 +117,7 @@ export type AnyNumber = number | bigint;
  * behavior and interaction with the Aptos network
  */
 export type AptosSettings = {
-  readonly network: Network;
+  readonly network?: Network;
```

Comment:
Why is network optional?

Reply:
good catch. So technically `network` shouldnt be required to pass in when initializing `AptosConfig` as `Aptos` resolves to a default network - Devnet. It was working fine by now because the `args` `AptosConfig` accepts is optional.

I guess discussion on whether we should default to a network makes sense or not, but if `network` is required, when want to support a custom client, need to
```
const config = new AptosConfig({ network:Network.DEVNET, client: { provider: customClient } });
```

Reply:
I definitely do like being able to do `new AptosConfig();`, but maybe it's not explicit enough...